framework-cloud/framework-openshift-operator/src/main/java/org/kie/cloud/openshift/operator/scenario/OpenShiftOperatorScenario.java

```diff
@@ -60,33 +58,34 @@ private void deployOperator() {
         }
     }
 
-    private void createImageStreamsInOpenShiftProject() {
-        List<ImageStream> imageStreams = OpenShiftUtils.admin().client().imageStreams().inNamespace("openshift").list().getItems();
-        boolean kieServerImageStreamExists = imageStreams.stream().anyMatch(i -> i.getMetadata().getName().matches(".*kieserver-openshift"));
-        if (!kieServerImageStreamExists) {
-            logger.info("Creating image streams in 'openshift' project from " + OpenShiftConstants.getKieImageStreams());
-            try {
-                KubernetesList resourceList = OpenShiftUtils.admin().client().lists().load(new URL(OpenShiftConstants.getKieImageStreams())).get();
-                OpenShiftUtils.admin().client().lists().inNamespace("openshift").create(resourceList);
-            } catch (MalformedURLException e) {
-                throw new RuntimeException("Malformed URL of Kie image stream.", e);
-            }
-        }
-    }
-
     private void createCustomResourceDefinitionsInOpenShift() {
         List<CustomResourceDefinition> customResourceDefinitions = OpenShiftUtils.admin().client().customResourceDefinitions().list().getItems();
-        boolean operatorCrdExists = customResourceDefinitions.stream().anyMatch(i -> i.getMetadata().getName().equals("apps.kiegroup.org"));
+        boolean operatorCrdExists = customResourceDefinitions.stream().anyMatch(i -> i.getMetadata().getName().equals("kieapps.app.kiegroup.org"));
         if(!operatorCrdExists) {
             logger.info("Creating custom resource definitions from " + OpenShiftResource.CRD.getResourceUrl().toString());
-            OpenShiftBinaryClient.getInstance().executeCommand("CRD failed.", "create", "-f", OpenShiftResource.CRD.getResourceUrl().toString());
+            OpenShiftBinaryClient instance = OpenShiftBinaryClient.getInstance();
+            try (ProcessExecutor executor = new ProcessExecutor()) {
+                executor.executeProcessCommand(instance.getOcBinaryPath().toString() + " --config=" + instance.getOcConfigPath().toString() + " create -n " + getNamespace() + " -f " + OpenShiftResource.CRD.getResourceUrl().toString());
+            }
         }
     }
 
+    private void createServiceAccountInProject(Project project) {
```

Comment:
Similar operations are in APB. It would be good to move it in framework parent and share this functionality in both places.
For now we can leave it here and open a new PR with changes for creating Service acc and role.

Reply:
I wouldn't share it explicitly as the Operator installation can change. Current approach is using direct installation into the project, but in the future we may switch to installation using OLM.

Reply:
@sutaakar agreed on OLM and, honestly, that should probably be the main (only) install method tested here... since it will be the only supported method for users.